src/plugins/plot/spectralAggregatePlot/SpectralAggregateView.vue

```diff
@@ -21,15 +21,314 @@
 -->
 
 <template>
-<div>
-
+<div class="c-spectral-aggregate-plot-view gl-plot plot-legend-bottom"
+     :class="{'plot-legend-expanded': legendExpanded, 'plot-legend-collapsed': !legendExpanded }"
+>
+    <SpectralAggregatePlot ref="spectralAggregatePlot"
+                           class="c-spectral-aggregate-plot__plot-wrapper"
+                           :data="visibleData"
+                           :plot-axis-title="plotAxisTitle"
+                           :legend-expanded="legendExpanded"
+    />
 </div>
 </template>
 
 <script>
+import * as SPECTRAL_AGGREGATE from './SpectralAggregateConstants';
+import ColorPalette from '../lib/ColorPalette';
+import objectUtils from 'objectUtils';
+import SpectralAggregatePlot from './SpectralAggregatePlot.vue';
 
 export default {
-    inject: ['openmct', 'domainObject']
+    components: {
+        SpectralAggregatePlot
+    },
+    inject: ['openmct', 'domainObject'],
+    data() {
+        return {
+            colorMapping: {},
+            composition: {},
+            currentDomainObject: this.domainObject,
+            isRealTime: (this.openmct.time.clock() !== undefined),
```

Comment:
If we're not using isRealTime in the markup above, or anywhere else in the code, we should remove this property.